modules/admin-ui/src/main/webapp/styles/views/modals/_hotkey-cheat-sheet.scss

```diff
@@ -0,0 +1,19 @@
+td.hotkey {
+    text-align: center;
+}
+
+// TODO This needs to be more specific!
```

Comment:
What do you mean by "needs to be more specific"? I suggest to either a) just remove the TODO comment, b) make it more specific and remove the TODO comment or c) create a JIRA issue and remove the TODO comment.

You see, I don´t like TODO comments ;-)

Reply:
I don't either (well, at least in finished and published commits) and this was totally an oversight probably due to prolonged exposure to high temperatures. :wink: